src/main/java/org/terasology/fluid/system/FluidAuthoritySystem.java

```diff
@@ -91,7 +92,7 @@
      * @param itemComponent  A component included for filtering out non-matching events. Here, we only want entities
      *                       which are used as items.
      */
-    @ReceiveEvent
+    @ReceiveEvent(priority= EventPriority.PRIORITY_HIGH)
```

Comment:
The default handlers for an event should, if possible, not be set to high priority. Reason being is that other systems should have the chance to interact with the event before the default handler does it job. 

I like to cite the health and damage system as an example. An initial damage event might be multiplied by some poisoning system, and then reduced by some armor system. The default handler will "just" apply whatever damage is left in the end.
In general, if registering a new system with a priority that's higher than "PRIORITY_LOW" I would have the assumption that the default handler will kick in afterwards.

Reply:
What do you propose I do then? 
If I do not increase the priority, both the methods fillFluidContainerItem and drinkConsumed collide and are called simultaneously which leads to the container not being filled as desired.

Reply:
The constants in `EventPriority` are just numbers (see https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/entitySystem/event/EventPriority.java). 

I don't know the priority of the other event handler, but maybe LOW and NORMAL would also work, no need to use HIGH. You could also use `LOW+10` or similar.

Reply:
SO is NORMAL+10 alright? low and normal would require a change in the thirst module too. I could do that too.. whichever you think is better.

Reply:
If LOW would work for the thirst module that sounds better. Might be that there was some reason it was put to some higher value, but I guess somebody just thought that NORMAL would be, well, "normal" and the right priority to put in the first place.

Reply:
Done.
Thirst PR: https://github.com/Terasology/Thirst/pull/14